### DISCUSSION TITLE
Repository is now public

### DISCUSSION THREAD

**Discussion by iluvcapra** [iluvcapra/py-ptsl#4] [Announcements]
I've gone through the repository history and deleted the PTSL.proto file in order to make the repository public and not run afoul of the PTSL EULA.

If you've cloned the repository prior to now you'll probably have to re-clone the repo with all of the amended commits.

I'm also considering dropping all collaborators from the project as I did this mostly to share the code in a private way and I haven't had any engagement or PRs; if you're interested in contributing to this project in the future please reach out to me otherwise I'll be removing collaborators in the next day or two.

@alubin620 @samueljustice @chigkim @jordanalexsilva @AdamVenn @geohook @ppstam @marcuswuest 
@FigBug @pneyrinck @patrickcusack @jharaldson @will-riley @BobBrownConsulting

**Comment by samueljustice** [comment]
Thanks for the heads up!
Samuel Justice
Sound Supervisor/Co-Founder

SWEET JUSTICE SOUND
Office Address: The Old Stables, Eccelsden Lane, Angmering, BN16 4DQ, United Kingdom
P: 01903 958037 Ext. 301
M: 07468 497479
www.sweetjusticesound.com <http://www.sweetjusticesound.com/>

VAT: GB243641910
Company No: 09670591

**Comment by jharaldson** [comment]
Went down the path of trying to do things with AAF files (pyaaf2) due to limited support in Protools. A topic on its own though...
Very nice repo!

**Comment by marcuswuest** [comment]
Hi Jamie,
Ok, thanks for the info!
Ciao,
Marcus

**Comment by alubin620** [comment]
Hi Jamie,

Thanks for informing us – I'm still interested in contributing, this project has slipped my mind unfortunately but I will clone it and check it out within the next few days.

Thanks!

**Reply by iluvcapra** [reply]
When you have a contribution please reach out! There’s a lot that needs to be cleaned-up now that it’s public.
